lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -15,13 +15,14 @@ sub new {
     my ($class, $args) = @_;
     return bless {
         grub_menu_base => $args->{grub_menu_base},
-        tag_first_entry_highlighted => 'grub-menu-agama-installer-highlighted',
+        tag_agama_installer_highlighted => 'grub-menu-agama-installer-highlighted',
     }, $class;
 }
 
 sub expect_is_shown {
     my ($self) = @_;
-    assert_screen($self->{tag_first_entry_highlighted}, 60);
+    send_key_until_needlematch($self->{tag_agama_installer_highlighted}, 'down');
```

Comment:
this methods refers to "expect is shown the grub" and should just expect that is shown in some way, further action like selecting an entry should go in other methods.